src/features/classification/Classification.js

```diff
@@ -28,22 +28,24 @@ type Props = {
     modifiedBy?: string,
     name?: string,
     onClick?: (event: SyntheticEvent<HTMLButtonElement>) => void,
+    useAppliedByLabel?: boolean,
```

Comment:
Can flip to `useImportByLabel` boolean instead (since that by default would be false) -- would that be preferred?

Reply:
Would it make sense to name it something like `isImportedClassification` so that it's not necessarily tied to the label? I think that way we can avoid the consumer of the component having to know when it is appropriate to use each label, or choosing a label incorrectly.

We could also use an enum if we expect that there could be more cases later on.